lib/Config.py

```diff
@@ -82,3 +84,19 @@ def getFile(cls, getfile, unpack=True):
           if len(fzip.namelist())>0:
             data=BytesIO(fzip.read(fzip.namelist()[0]))
     return (data, response)
+  
+  # Get Microsoft Bulletin
```

Comment:
Hi,
it's possible, i chose this to separate download file function and read data from xlsx file, like the first plugin of MS.